waves/src/App.tsx

```diff
@@ -302,7 +297,15 @@ function App() {
                                 <Text textStyle="info">1 BTC ~ {rate.ask} USDT</Text>
                             </Box>
                             <Box>
-                                {button}
+                                <Button
+                                    onClick={makeNewSwap}
+                                    size="lg"
+                                    variant="main_button"
+                                    isLoading={isCreatingNewSwap}
+                                    disabled={disableSwapButton}
```

Comment:
I'd prefer naming of these variables in terms of states instead of actions, e.g. `isSwapButtonDisabled`.